core/server/data/migrations/versions/4.0/25-populate-members-paid-subscription-events-table.js

```diff
@@ -29,6 +29,14 @@ module.exports = createTransactionalMigration(
             if (interval === 'month') {
                 return amount;
             }
+
+            if (interval === 'week') {
+                return amount * 4;
+            }
+
+            if (interval === 'day') {
+                return amount * 30;
+            }
         }
```

Comment:
@allouis So we've seen this migration fail before where the subscription interval was `week` and obviously this PR fixes that, but would it be useful to have some sort of error/ignoring for when it's still not handled? I realise we're probably handling all cases now but..... 😅

Reply:
So the only valid values from stripe are: "day, week, month or year." - but I think you're right, we should probably error if we encounter an unknown one

Reply:
I foresee `decade` or something 😆